### DISCUSSION TITLE
What goes under PHOTOPRISM_STORAGE_PATH (/photoprism/storage)

### DISCUSSION THREAD

**Discussion by PawelTroka** [photoprism/photoprism#1411] [Q&A] [answered]
in official docs it is written that it is a storage PATH for cache, database and sidecar files

however I am interested in what folders are under this directory, so I can mount cache on a fast external medium-sized SSD drive, and configuration on a small fast internal SSD drive

**Comment by PawelTroka** [comment, marked as answer]
ok, from https://github.com/photoprism/photoprism/blob/dcacd28e6e6f559d4188958a8fbe0f88331d284c/Makefile and https://github.com/photoprism/photoprism/issues/556

I think it is those 3:
```
/photoprism/storage/config
/photoprism/storage/cache
/photoprism/storage/sidecar
```

**Comment by lastzero** [comment]
Yes.

**Comment by lastzero** [comment]
You can also configure a different cache path by the way, it doesn't need to be in storage. See config options.

**Comment by PawelTroka** [comment]
thanks!

**Comment by PawelTroka** [comment]
btw which one will grow the biggest? assuming I have 1 TB of originals, how much can I expect each of them to grow?

**Comment by PawelTroka** [comment]
ok, from:
https://docs.photoprism.org/getting-started/faq/
`Most space is consumed by thumbnails: These are high-quality resampled, smaller versions of your originals.`

**Comment by lastzero** [comment]
You can reduce the JPEG quality and (pre-rendered) size to save storage, depending on your use case and expectations. Disk space is extremely cheap nowadays, so I personally recommend going for quality and performance even if that means 30-40% overhead. Do the math how much it costs extra, compared to your camera equipment and server hardware.
